test/config/stack_config.json

```diff
@@ -64,6 +64,23 @@
       }
     }
   },
+  "docker_local_run":{
+    "rackhd_host": "localhost",
+    "type": "linux",
+    "ucsm_ip" : "172.31.128.252",
+    "ucs_service_uri" : "https://localhost:7080",
+    "ucsm_config_file" : "./tests/ucs/ucsm_config.xml",
+    "install-config": {
+      "ports": {
+        "mongo_port": 27017,
+        "http": 9090,
+        "https": 8443,
+        "amqp_ssl": 5672,
+        "amqp_nossl": 5671,
+        "ssh": 22
```

Comment:
Thank you for your review!
"22" is not the ssh-port to reach the docker container, it's the ssh-port to log into the host which is running the container.
And I'm running FIT with below command and it works well :
```
python run_tests.py -test tests -group smoke -stack docker_local_run --sm-amqp-url amqp://guest:guest@localhost:5672 -v 4 -xunit
```

From my understanding , the reason for FIT and CIT to log into the rackhd host  is that they need to run commands like: ```ipmitool, rabbitmqctl```
And RackHD depends on the tools ipmitool and rabbitmq server.
FIT and CIT can reuse them.

But I don't think it's a good idea to run smoke test of RackHD depending  on the environment of RackHD.
FIT and CIT should run with only the access of the API , such as http://localhost:9090, amqp://guest:guest@localhost:5672.
About ipmitool, maybe we can setup a docker image for FIT and CIT which installed ipmitool.

 If we really want to log into a container with "ssh", we need to install a ssh server in the container. But I just don't think it's a good idea. There is a document ["If you run SSHD in your Docker containers, you're doing it wrong!"](https://jpetazzo.github.io/2014/06/23/docker-ssh-considered-evil/
)